src/opm/output/eclipse/Summary.cpp

```diff
@@ -459,6 +459,14 @@ double efac( const std::vector<std::pair<std::string,double>>& eff_factors, cons
     return (it != eff_factors.end()) ? it->second : 1;
 }
 
+inline quantity alqrate( const fn_args& args ) {
+    const auto& well = args.schedule_wells.front();
+    const auto& name = well.name();
+    if( args.wells.count( name ) == 0 ) return { 0, measure::rate };
+    const auto alq_value = args.wells.at(name).rates.get(rt::alq, 0.0);
+    return { alq_value, measure::rate };
```

Comment:
This looks suspicious to me.  `Measure::rate` is a reservoir volume rate.  Is that always the correct choice?  In the context of `xGLIR`, I'd think [`measure::gas_surface_rate`](https://github.com/OPM/opm-common/blob/6ee4e5ce9a8ca272bb91281ee7e28bef6b305998/opm/parser/eclipse/Units/UnitSystem.hpp#L57) would be more appropriate here.

On another note, is there some mechanism that will assign these values from the simulator side?  If not, this will report 0.0 for all wells.  Finally, you could also probably add the field and group-level reporting here by putting
```
{ "GGLIR", alqrate },
{ "FGLIR", alqrate },
```
into the processor function table.

Reply:
> is there some mechanism that will assign these values from the simulator side?

Right, the plan is to assign these from `WellStateFullyImplicitBlackoil.hpp` in `opm-simulators` in a future PR.

Reply:
> Right, the plan is to assign these from `WellStateFullyImplicitBlackoil.hpp` in `opm-simulators` in a future PR.

Do you have a time estimate for when you'll submit that PR?  We're getting closer to the next OPM release (designation 2020.10) and I'd prefer not having a partially complete feature that unconditionally outputs 0.0 to the summary files.

Reply:
I will try to submit it this week.